### DISCUSSION TITLE
Filter by Cloud Service Provider

### DISCUSSION THREAD

**Discussion by iamabrom** [ilyas-it83/CloudComparer#211] [Ideas]
Hello! I hope everyone is well!

Any thoughts on creating a filtering option towards the top of the page to select only the CSPs you wish to compare? This would also make it easier to view on a mobile device.

Thoughts?

**Comment by dna-2** [comment]
i would hope u able to show the price of respective cloud component. 

great effort! Tks.

**Comment by nokhiz** [comment]
good idea! I just wanted the same... a filter would be really good.

**Reply by rockync** [reply]
+1 for filter. I just want to compare GCP, Azure and AWS and would like to maximize the real estate for these 3. Awesome work.
